_inc/client/traffic/related-posts.jsx

```diff
@@ -113,19 +113,25 @@ export const RelatedPosts = moduleSettingsForm(
 								{
 									[
 										{
-											url : '1-wpios-ipad-3-1-viewsite.png',
+											url: '1-wpios-ipad-3-1-viewsite.png',
 											text: __( 'Big iPhone/iPad Update Now Available' ),
-											context: __( 'In "Mobile"' )
+											context: __( 'In "Mobile"', {
+												comment: 'It refers to the category where a post was found. Used in an example preview.'
```

Comment:
I got this from
https://github.com/Automattic/i18n-calypso#more-translate-examples
specifically this one:
```js
// add a comment to the translator
var content = i18n.translate( 'g:i:s a', {
        comment: 'draft saved date format, see http://php.net/date'
    } );
```